### DISCUSSION TITLE
I set my JWT expiry to 604800 (one week), but I'm getting logged out after a day

### DISCUSSION THREAD

**Discussion by bradgreens** [supabase/supabase#6675] [Questions] [answered]
Just as the title says.

As many examples suggest, I am using a server-side endpoint to call `supabase.auth.api.setAuthCookie(request, response)` and fetch the auth session with the server-side cookie. 

I think this means the Auth Setting for JWT Expiry is not taken into account here. What might I be doing wrong? If the server-side cookie expires more quickly, do I need to also check client-side which would incorporate the localStorage/JWT session data?

It looks to me like `setAuthCookie` is supposed to pull `maxAge` from the request data on the `SIGN_IN` event.

**Comment by GaryAustin1** [comment, marked as answer]
https://github.com/supabase/gotrue-js/search?q=lifetime
Appears cookie is 8 hours default setting.
Edit: https://github.com/supabase/gotrue-js/issues/229
